src/cider/nrepl/middleware/inspect.clj

```diff
@@ -87,4 +112,7 @@
    :expects #{"eval"}
    :handles {"inspect-pop" {}
              "inspect-push" {}
-             "inspect-refresh" {}}}))
+             "inspect-refresh" {}
```

Comment:
Btw, while you're on this you can add the missing metadata here (op description & params).